pkg/gardenlet/bootstrap/util/util.go

```diff
@@ -105,6 +106,42 @@ func UpdateGardenKubeconfigSecret(ctx context.Context, certClientConfig *rest.Co
 	return kubeconfig, nil
 }
 
+// UpdateGardenKubeconfigCAIfChanged checks if the garden cluster CA given in the gardenClientConnection differs from the CA in the kubeconfig secret
+// and updates the secret to contain the new CA if that's the case.
+func UpdateGardenKubeconfigCAIfChanged(ctx context.Context, seedClient client.Client, kubeconfigAsBytes []byte, gardenClientConnection *config.GardenClientConnection) ([]byte, error) {
+	if kubeconfigAsBytes == nil {
+		return nil, fmt.Errorf("no kubeconfig given")
+	}
```

Comment:
```suggestion
	}

```

(please introduce similar newlines to improve readability)